src/main/java/teammates/common/util/RichTextSecurity.java

```diff
@@ -0,0 +1,30 @@
+package teammates.common.util;
+
+import org.owasp.html.HtmlPolicyBuilder;
+import org.owasp.html.PolicyFactory;
+
+public final class RichTextSecurity {
+
+    private static PolicyFactory policy = new HtmlPolicyBuilder()
+                    .allowStandardUrlProtocols()
+                    .allowAttributes("title").globally()
+                    .allowAttributes("href").onElements("a")
+
+                    .allowAttributes("align")
+                        .matching(true, "center", "left", "right", "justify", "char")
+                        .onElements("p")
+                    .allowElements(
+                        "a", "p", "div", "i", "b", "em", "blockquote", "tt", "strong",
+                        "br", "ul", "ol", "li")
+                    .allowElements("quote", "ecode")
+                    .toFactory();
+
+    private RichTextSecurity() {
+
+    }
+
+    public static String sanitize(String content) {
+        return policy.sanitize(content);
```

Comment:
Let's put all these in the `Sanitizer` class, perhaps in a new method called `sanitizeForRichText`. Don't forget to add header comment and unit test for that method.